### PR TITLE
PostProcessing: Use Dubois algorithm for anaglyph shader.

### PR DESCRIPTION
**Pull request by Armada651**
This algorithm provides a better retention of color when using Anaglyph Red-Cyan glasses.

### CODE REVIEW

**Comment by Armada651** [comment]
@degasus This PR drastically increases Anaglyph quality, could you review it?

I could also use matrix declarations here instead of vectors if that is preferred.

File: Source/Core/VideoBackends/OGL/PostProcessing.cpp [hunk 1]
```diff
@@ -36,6 +36,26 @@ static char s_vertex_shader[] =
 	"	uv0 = rawpos * src_rect.zw + src_rect.xy;\n"
 	"}\n";
 
+// Anaglyph Red-Cyan shader based on Dubois algorithm
+// Constants taken from the paper:
+// "Conversion of a Stereo Pair to Anaglyph with
+// the Least-Squares Projection Method"
+// Eric Dubois, March 2009
+static char s_anaglyph_shader[] =
```
Comment (degasus) [inline Source/Core/VideoBackends/OGL/PostProcessing.cpp]: const

**Comment by degasus** [comment]
The matrix way LGTM

@dolphin-emu-bot allowmerge

File: Source/Core/VideoBackends/OGL/PostProcessing.cpp (hunk 1 above)
Comment (Armada651) [inline Source/Core/VideoBackends/OGL/PostProcessing.cpp]: The other shaders are also not const, I'll mark them too.

**Pull request merged**